### DISCUSSION TITLE
how to use projects plugin

### DISCUSSION THREAD

**Discussion by verajosemanuel** [LazyVim/LazyVim#2507] [Q&A]
how to add a folder to the list of projects to appear ?

As documentation states, all .git folders are added by default.

I have activated the **Projects** Extra and Telescope Projects seems to work but project list is still empty
![image](https://github.com/LazyVim/LazyVim/assets/15836677/77efd486-d8eb-4960-976a-9cced826e505)

Telescope projects is empty too:
<img width="955" alt="image" src="https://github.com/LazyVim/LazyVim/assets/15836677/6d1c7970-4b1d-4eca-b994-1ca389cb18da">

  how to add a folder to telescope list of sessions?
 how to create/remove/list all the sessions/projects I have?

Thanks in advance

**Comment by dpetka2001** [comment]
See #2473 and corresponding keybindings of the plugin in the plugin's repo README.

**Comment by verajosemanuel** [comment]
Thanks @dpetka2001  for your answer, but the link you provided was read a couple of days ago and after reading again seems a bit difficult for me to guess how to use the plugin 
It mentions some configuration items or auto commands regarding the plugin but not a specific how to. 
So, I tried a bunch of ideas and  discovered a way of solving my issue, surely a bit convoluted, but I leave it here for future users searching for help.

1. Activate projects plugin
2. Cd to any folder you consider a project and open neovim
3. Leader + s  + C
4. Filter commands searching for the word “add”
5. Add project is available, select and press enter
6. Now open cmd line and  execute ‘Telescope projects’ 
7. You will see a list of the added folder

**Comment by StefanoChiodino** [comment]
[#1978](https://github.com/LazyVim/LazyVim/discussions/1978)
